index.html

```diff
@@ -271,7 +270,7 @@
       <p>Change the logical document display to the specified URI of a logical document.</p>
       <p>Don't forget to include the "dot extension" ie .json.</p>
 
-      <b class="enterInstruct">Enter <span class="enterInstruct" class= "definition" title = "URI (Uniform Resource Identifier): A string of characters used to identify a name of a resource">URI:</span></b><br>
+      <b class="enterInstruct">Enter <span class= "definition" title = "URI (Uniform Resource Identifier): A string of characters used to identify a name of a resource">URI:</span></b><br>
```

Comment:
FYI, an element may have more than one class, as in: 

    <span class="enterInstruct definition" title="URI...">URI</span>

Looks like you were eliminating a duplicated class here, so it looks like you're all set.